server/storage/mvcc/kvstore_txn.go

```diff
@@ -279,9 +279,9 @@ func (tw *storeTxnWrite) deleteRange(key, end []byte) int64 {
 }
 
 func (tw *storeTxnWrite) delete(key []byte) {
-	ibytes := newRevBytes()
-	idxRev := revision{main: tw.beginRev + 1, sub: int64(len(tw.changes))}
-	revToBytes(idxRev, ibytes)
+	ibytes := NewRevBytes()
+	idxRev := RevisionTombstone{Main: tw.beginRev + 1, Sub: int64(len(tw.changes))}
+	RevToBytes(idxRev, ibytes)
 
 	ibytes = appendMarkTombstone(tw.storeTxnCommon.s.lg, ibytes)
 
```

Comment:
What do you think about setting tombstone on RevisionTombstone struct, instead of setting it separetly via appendMarkTombstone?
```suggestion
	idxRev := RevisionTombstone{Main: tw.beginRev + 1, Sub: int64(len(tw.changes)), tombstone: true}
	RevToBytes(idxRev, ibytes)
```

Reply:
Yeah I like that